pkg/sqlite/load.go

```diff
@@ -135,8 +135,11 @@ func (s *SQLLoader) AddPackageChannels(manifest registry.PackageManifest) error
 	defer addReplaces.Close()
 
 	if _, err := addPackage.Exec(manifest.PackageName); err != nil {
```

Comment:
instead of this, can we just always "update" package channels, so that it creates if it's not there? (rather than trying to write data and falling back to update on failure?)

non-blocking, we can follow up.

Reply:
In the old format, it was not allowed to have packagename collisions and that's exactly the case in the new format where we are trying to update the db to contain a new version of some operator. I tried to take advantage of that such that when there are no previous versions of an operator in the db it defaults to the old format behavior